File: app/services/omni/order.rb
class Omni::Order::Service
  include Buildit::Service::Base

  service 'Order', 'Omni.service'

  connected_mode(Buildit::Service::Backend::Crud) do |config|
    config.model = Omni::Order
  end

end # class Omni::Order::Service

File: app/assets/javascripts/omni/store/Order.js
Ext.define('Omni.store.Order', {
  extend       : 'Ext.data.Store',
  alias        : 'store.omni-Order',
  model        : 'Omni.model.Order',
  autoLoad     : false,
  storeId      : 'OrderStore',
  remoteFilter : true,
  remoteSort   : true,

  constructor  : function (config) {
    var me    = this;
    me.callParent(this);
    
    var proxy = Ext.Object.merge({}, me.getProxy());
    me.setProxy(proxy);
  } // constructor

}); // Ext.define('Omni.store.Order'